container/spi/src/main/java/org/mobicents/slee/container/activity/ActivityContextCacheDataInterface.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2011, Red Hat, Inc. and individual contributors
+ * by the @authors tag. See the copyright.txt in the distribution for a
+ * full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.mobicents.slee.container.activity;
+
+import java.util.Map;
+import java.util.Set;
+
+import javax.slee.facilities.TimerID;
+
+import org.mobicents.slee.container.sbbentity.SbbEntityID;
+
+
+public interface ActivityContextCacheDataInterface {
```

Comment:
yes. I want to create FakeActivityContextCacheData. I tried to create the real one. But It related to jboss cache. Then I create the fakecache for testing.